packages/react-instantsearch/src/widgets/__tests__/__utils__/all-widgets.tsx

```diff
@@ -91,6 +91,9 @@ function Widget<TWidget extends SingleWidget>({
     case 'SearchBox': {
       return <widget.Component onSubmit={undefined} {...props} />;
     }
+    case 'FrequentlyBoughtTogether': {
+      return <widget.Component objectIDs={['1']} {...props} />;
+    }
```

Comment:
Nitpick: we can put this in common with the statement just below it.

```suggestion
    case 'FrequentlyBoughtTogether':
```